Formula/staticcheck.rb

```diff
@@ -0,0 +1,30 @@
+class Staticcheck < Formula
+  desc "State of the art linter for the Go programming language"
+  homepage "https://staticcheck.io/"
+  url "https://github.com/dominikh/go-tools/archive/2020.1.5.tar.gz"
+  sha256 "aa57272dd96a8d93af74c504c9a8f9949712e1983512830a0b82f3afbd96b4ac"
+  license "MIT"
+  head "https://github.com/dominikh/go-tools.git"
+
+  depends_on "go" => :build
```

Comment:
```suggestion
  depends_on "go"
```

Apparently `go` is needed at runtime.